analysis/quantifier-comparisons/12-spatial-transcriptomics.Rmd

```diff
@@ -0,0 +1,595 @@
+---
+title: "Spatial Transcriptomics Exploration"
+author: "Ally Hawkins for CCDL"
+output: 
+  html_notebook:
+    toc: true
+    toc_float: true
+---
+
+In this notebook we are comparing the use of Alevin-fry and Spaceranger for quantifying spatial transcriptomics libraries. 
+Two spatial transcriptomic libraries were quantified using Alevin-fry and Spaceranger and the results were combined following the [Alevin-fry tutorial](https://combine-lab.github.io/alevin-fry-tutorials/2021/af-spatial/). 
+We are comparing that to if we were to not integrate the Spaceranger data with Alevin-fry and only use Spaceranger. 
+
+Additionally in this notebook we also explore the use of storing spatial data in a [`SpatialExperiment` object](https://bioconductor.org/packages/release/bioc/vignettes/SpatialExperiment/inst/doc/SpatialExperiment.html#3_Common_operations) or in a [`Seurat` object](https://satijalab.org/seurat/articles/spatial_vignette.html). 
+
+## Set Up
+
+```{r}
+library(magrittr)
+library(ggplot2)
+library(SingleCellExperiment)
+library(SpatialExperiment)
+library(ggspavis)
```

Comment:
I'm noticing that `Seurat` is not loaded here, but functions later are called without `Seurat::`. I think I would probably leave Seurat unloaded if you can, but I might also remove `ggspavis` here. For following what is going on in an exploratory notebook, I find those prefixes very helpful.